src/main/scala/firrtl/Compiler.scala

```diff
@@ -324,8 +334,9 @@ trait Transform extends TransformLike[CircuitState] with DependencyAPI[Transform
       Dependency[SystemVerilogEmitter] :: Nil
 
     val emitters = inputForm match {
-      case C => Dependency[ChirrtlEmitter]      :: Dependency[HighFirrtlEmitter]   :: Dependency[MiddleFirrtlEmitter] :: lowEmitters
-      case H => Dependency[HighFirrtlEmitter]   :: Dependency[MiddleFirrtlEmitter] :: lowEmitters
+      case C =>
+        Dependency[ChirrtlEmitter] :: Dependency[HighFirrtlEmitter] :: Dependency[MiddleFirrtlEmitter] :: lowEmitters
```

Comment:
I think words per line limitation can be increased? Maybe 120 or 180?

Reply:
It's currently set to 120, I don't think we should go higher than that.